### ISSUE TITLE
AccessibilityHelper snippet lacks spaces

### ISSUE THREAD

**Issue by wjk** [MicrosoftDocs/xamarin-docs#2187]
It's very strange. Yet, when I cut/paste the snippet into my editor to fix it, the spaces show up there, exactly as intended! I don't understand how this could have happened. Inspection in the browser devtools reveals that the `data-author-content` attribute on the `&lt;code&gt;` tag lacks spaces. The span tags within the code block have spaces placed inconsistently.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 0dabb36b-d9cc-a56d-f27f-4109c246dd3a
* Version Independent ID: 33607786-ad8e-df94-a12f-9cdb32571741
* Content: [Accessibility on macOS - Xamarin](https://docs.microsoft.com/en-us/xamarin/mac/app-fundamentals/accessibility)
* Content Source: [docs/mac/app-fundamentals/accessibility.md](https://github.com/MicrosoftDocs/xamarin-docs/blob/live/docs/mac/app-fundamentals/accessibility.md)
* Product: **xamarin**
* Technology: **xamarin-mac**
* GitHub Login: @conceptdev
* Microsoft Alias: **crdun**

**Comment by conceptdev** [comment]
Hi @wjk, could you provide more info? The code block source looks like this:

```csharp
public static class AccessibilityHelper
{
    [System.Runtime.InteropServices.DllImport (ObjCRuntime.Constants.ObjectiveCLibrary)]
    extern static void objc_msgSend (IntPtr handle, IntPtr selector, IntPtr label);

    static public void SetAccessibilityLabel (this NSView view, string value)
    {
        objc_msgSend (view.Handle, new ObjCRuntime.Selector ("setAccessibilityLabel:").Handle, new NSString (value).Handle);
    }
}
```

and renders like this:

<img width="581" alt="Screen Shot 2019-12-11 at 2 56 35 PM" src="https://user-images.githubusercontent.com/199373/70667725-783fbf00-1c26-11ea-9356-c49fb5aa2973.png">

Could you clarify the problem you're experiencing? Thanks.

**Comment by wjk** [comment]
On my Mac (macOS Catalina 10.15.1), using Safari 13.0.3, the block in question renders like this:

<img width="714" alt="Screen Shot 2019-12-11 at 6 46 43 PM" src="https://user-images.githubusercontent.com/1915951/70670269-961d1c00-1c46-11ea-8ad5-288c0169ea85.png">

What browser and OS are you using? Also, why would the source block have been written like that in the first place? Thanks!

**Comment by conceptdev** [comment]
Yikes! I wonder if there is incorrect unicode space characters in use somehow.

Do you have the same experience on all 'code blocks' on docs.microsoft.com, or just this one?

**Comment by conceptdev** [comment]
UPDATE: it _does_ look like a weird character issue (latest commit shown below). No idea why it didn't reproduce for me locally. Hopefully when this is published it will resolve the issue. Thanks for reporting it!

![Screen Shot 2019-12-11 at 4 36 34 PM](https://user-images.githubusercontent.com/199373/70672564-8563aa80-1c34-11ea-8a04-8355baa8340b.png)
